tests/unit/fish_completion_test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+. ./tests/unit/utils.sh --source-only
```

Comment:
We have a helper for that, you can just use:

`include './tests/unit/utils.sh'`